tests/test_wave_03.py

```diff
@@ -54,13 +54,13 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
-
-    raise Exception("Test needs to be completed.")
+    assert HORROR_1 and INTRIGUE_3 and FANTASY_4 in friends_unique_movies
```

Comment:
This statement does not do what you think it does! It checks if `HORROR_1` is truthy, if `INTRIGUE_3` is truthy, and then checks to see if `FANTASY_4` is in `friends_unique_movies`. Each part of the `and` is its own statement. This statement should be:
```
assert HORROR_1 in friends_unique_movies and INTRIGUE_3 in friends_unique_movies and FANTASY_4 in friends_unique_movies
```
Given how long that statement is, it'd be best to have each one be its own assert.